osiris/authentication/models/accounts.go

```diff
@@ -42,6 +43,10 @@ func (account *Account) Validate() (map[string]interface{}, bool) {
 	temp := &Account{}
 
 	//check for errors and duplicate emails
+
+	db := GetDB()
+	fmt.Println("DB DEBUG")
```

Comment:
We want to do logrus (logging) instead of print statements. See "How to build a microservice" for that.

Reply:
Actually there shouldn't be any logging or printing ast all. Gonna fix that.